pkg/controller/vitessshard/reconcile_tablets.go

```diff
@@ -231,6 +249,8 @@ func (r *ReconcileVitessShard) reconcileTablets(ctx context.Context, vts *planet
 	})
 	if err != nil {
 		resultBuilder.Error(err)
+	} else {
```

Comment:
Leftover?

Reply:
Oh weird, I was sure I had removed this! Yes, definitely. Removing.

Reply:
Removed.